draft-ietf-quic-transport.md

```diff
@@ -1198,12 +1198,25 @@ The fields of a GOAWAY frame are as follows:
 
 # Packetization and Reliability {#packetization}
 
-The maximum packet size for QUIC is the maximum size of the encrypted payload of
-the resulting UDP datagram.  All QUIC packets SHOULD be sized to fit within the
-path's MTU to avoid IP fragmentation.  The recommended default maximum packet
-size is 1350 bytes for IPv6 and 1370 bytes for IPv4.  To optimize better,
-endpoints MAY use PLPMTUD {{!RFC4821}} for detecting the path's MTU and setting
-the maximum packet size appropriately.
+The maximum packet size for QUIC is the maximum size of the entire UDP payload,
+including the public header, encrypted payload, and any authentication fields.
+All QUIC packets SHOULD be sized to fit within the path's MTU to avoid IP
+fragmentation. To optimize bandwidth efficiency, endpoints MAY use Path MTU
+Discovery ({{!RFC1191}}, {{!RFC1981}}) or Packetization Layer Path MTU
+Discovery ({{!RFC4821}}) for detecting the path's MTU, setting the maximum
+packet size appropriately, and storing the result of previous PMTU
+determinations. In the absence of these mechanisms,  the recommended default
+maximum packet size is 1350 bytes for IPv6 and 1370 bytes for IPv4. 
+
+QUIC endpoints that implement any kind of MTU discovery SHOULD maintain a
+separate PMTU estimate for each IP address the peer is using in the
+connection. Endpoints SHOULD maintain an estimate for each combination of
+local and remote IP addresses (as each pairing may have a different minimum
+MTU in the path).
+
+All handshake packets MUST include enough PADDING frames to bring the packet
```

Comment:
+1, except that the transport draft defines a PADDING frame currently as simply the Frame Type 0, meaning that a PADDING frame is basically 1 byte long. I'll change this to be Type + 0-padding to the end of the packet.